kuma/contributions/jinja2/contributions/contribute.html

```diff
@@ -0,0 +1,228 @@
+{% block document_head %}
+    {% with form=form, is_popover=False %}
+        {% include "contributions/includes/contribution-form.html" %}
+    {% endwith %}
+{% endblock %}
+
+{% block contrib_popup %}{% endblock %}
+
+{% block content %}
+
+<div id="contributions-page" class="conributions-page">
+    <div class="section">
+        <img src="{{ static('img/hero-dino-blank.png') }}" class="backdrop-image" alt="" role="presentation">
+        <div class="index">
+            <h2>{{_('FAQs')}}</h2>
+        </div>
+        <ul class="faqs clear">
+            <li class="faq">
+                <h3>{{_('Why is MDN asking me for money?')}}</h3>
+                <span class="faq-number">1</span>
+                <p>
+                    {% trans url="https://hacks.mozilla.org/2018/01/introducing-the-mdn-product-advisory-board/" %}
+                        In 2017, MDN established a
+                        <a href="{{ url }}">Product Advisory Board</a> and
+                        added documentation from several other developer
+                        programs to begin to create a truly world-wide,
+                        web-wide developer resource. Having this broader
+                        readership contribute directly to MDN creates a
+                        tighter, healthier loop between our audience (you), our
+                        content (written for you), and our financial supporters
+                        (also, you, again, if you can help by putting a price
+                        on a priceless resource!).
+                    {% endtrans %}
+                </p>
+                <p>
+                    {% trans url="https://developer.mozilla.org/docs/MDN/About#History_of_MDN_Web_Docs" %}
+                        MDN <a href="{{ url }}">originally began as the Mozilla Developer Network</a>
+                        and was used almost exclusively for Firefox internal
+                        documentation. But we have outgrown that. And your needs
+                        have outgrown that.  We’ve now broadened our coverage to
+                        web standards beyond Firefox, and our funding should match
+                        that reality.
+                    {% endtrans %}
+                </p>
+                <span class="small">{{_('Did this answer your question?')}}</span>
+                <div class="vote-button-group">
+                    <button class="thumbs-up" data-faq="1">{% include 'includes/icons/emojis/thumbs-up.svg' %}</button>
+                    <button class="thumbs-down" data-faq="1">{% include 'includes/icons/emojis/thumbs-down.svg' %}</button>
+                </div>
+            </li>
+            <li class="faq">
+                <h3>{{_('If I just donate to the Mozilla Foundation, will that help MDN?')}}</h3>
+                <span class="faq-number">2</span>
+                <p>
+                    {% trans %}
+                        Not as likely and not as much. Donations to Mozilla
```

Comment:
"Donations to Mozilla" => "Donations to the Mozilla"

Reply:
Should be fixed in latest PR